### PR TITLE
Update documentation for enable_key_list_policy

### PR DESCRIPTION
**Pull request by tundeaoni**
The current suggests the option expects a string of either "enabled" or "disabled" but this results in an error `'acl.enable_key_list_policy' expected type 'bool', got unconvertible type 'string', value: 'enabled'`. Setting to a boolean value resolves this, also had a quick look at the code (https://github.com/hashicorp/consul/blob/d2b58cd0d6e84f04eed08d233b8e8b396f6d7663/agent/config/runtime.go#L109) and it suggests this too

### CODE REVIEW

**Comment by blake** [comment]
Hi @tundeaoni, thanks for fixing the docs for this parameter. I can merge this PR after you sign the CLA.

**Comment by tundeaoni** [comment]
@blake thanks, I have signed the CLA

**Review by blake — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/428503.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit ad07ed705c99958fc9c617c96509860f5c84c56d onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit ad07ed705c99958fc9c617c96509860f5c84c56d onto `release/1.10.x` succeeded!

**Pull request merged**
